astropy/io/fits/hdu/hdulist.py

```diff
@@ -921,9 +921,7 @@ def flush(self, output_verify="fix", verbose=False):
                     shutil.copy(filename, backup)
                 except OSError as exc:
                     raise OSError(
-                        "Failed to save backup to destination {}: {}".format(
-                            filename, exc
-                        )
+                        f"Failed to save backup to destination {filename}: {exc}"
```

Comment:
These days one should write this as `raise OSError(...) from exc` but probably out of scope here.

Reply:
I went ahead and added the `from exc` here. Feel free to tell me to revert.